behavior_tests/src/bt-helper-function-dir/do_test.py

```diff
@@ -0,0 +1,31 @@
+# ====------ do_test.py---------- *- Python -* ----===##
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+#
+# ===----------------------------------------------------------------------===#
+import subprocess
+import platform
+import os
+import sys
+
+from test_utils import *
+
+def setup_test():
+    change_dir(test_config.current_test)
+    return True
+
+def migrate_test():
+
+    call_subprocess(test_config.CT_TOOL +" --helper-function-dir")
+    helper_function_dir_root = os.path.abspath(os.path.join(get_ct_path(), os.pardir , os.pardir , "include"))
+    print("Helper functions directory: ", helper_function_dir_root)
+    return is_sub_string(helper_function_dir_root, test_config.command_output)
```

Comment:
Could you explain why that "if helper_function_dir_root is a substring of test_config.command_out, helper_function_dir_root is an expected output directory?"

Reply:
The `test_config.command_output` contains the output of `dpct --helper-function-dir` as well as some newlines appended extra to it. To test whether command output is helper function directory, I am checking whether it is a full substring in the command output.

I followed the examples of: 1) bt-version: https://github.com/oneapi-src/SYCLomatic-test/blob/SYCLomatic/behavior_tests/src/bt-version/do_test.py#L26

and 2) bt-help: https://github.com/oneapi-src/SYCLomatic-test/blob/SYCLomatic/behavior_tests/src/bt-help/do_test.py#L23

Reply:
could you show a example value of:  helper_function_dir_root and  test_config.command_output?